apps/cli/src/commands/build.ts

```diff
@@ -419,29 +362,60 @@ Update your application Dockerfile using one of the templates at https://github.
                 appGnutarPath,
             );
 
+            // prepare OCI Bundle
+            const extract = tar.extract();
+            const pack = tar.pack();
+            const appGnuTarStream = fs.createReadStream(appGnutarPath);
+            const appOCIBundleStream = fs.createWriteStream(appOCIBundlePath);
+            const ociConfigJSON = JSON.stringify(createConfig(imageInfo));
+
+            // add config.json
+            pack.entry({ name: "config.json" }, ociConfigJSON, function (err) {
+                if (err) throw err;
+                console.log("ERROR config.json");
+                pack.finalize();
```

Comment:
this looks out of place. because you will add more entries (for each gnuTar entry)

Reply:
I did some changes in a fixup, but the `.cartesi/app.ocibuindle.tar` is still incomplete.

I don't fully understand how these streams are supposed to work, something is missing to close/finish the tar, since I never hit the `console.log()` at lines 388 and 392.

But `.cartesi/app.ocibuindle.tar` has `config.json`  and `rootfs/` prefix there.